spec/jobs/private_body_decrypt_job_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe PrivateBodyDecryptJob, type: :job do
+  let!(:extra_fields) { create(:awesome_proposal_extra_fields, proposal: create(:extended_proposal)) }
+
+  before do
+    extra_fields.private_body = { "en" => '<xml><dl><dt name="something">Something</dt></dl></xml>' }
+    extra_fields.save!
+
+    allow(Rails.logger).to receive(:info)
+  end
+
+  it "logs and updates the decrypted body if extra fields are present" do
```

Comment:
Prefer to move test when proposals must be updated in a `context` clause like 
`when "proposals have no decrypted body"`

and include in it 
```
before do
  extra_fields.update_columns(decrypted_private_body: nil)
end
```

And then test if proposal is well updated